Base/usr/share/man/man2/scheduler_set_parameters.md

```diff
@@ -0,0 +1,55 @@
+## Name
+
+scheduler_set_parameters, scheduler_get_parameters - Set and get scheduler parameters for processes and threads
+
+## Synopsis
+
+```**c++
+int scheduler_set_parameters(SC_scheduler_parameters_params const* parameters);
+int scheduler_get_parameters(SC_scheduler_parameters_params* parameters);
```

Comment:
These aren't userspace-callable functions, are they? The only actual API that programs can use is still `sched_getparam`/`sched_setparam` and `pthread_getschedparam`/`pthread_setschedparam` as far as I can tell.

Reply:
I don't see a problem with documenting syscalls, although we would additionally need the POSIX API documentation.

Reply:
> I don't see a problem with documenting syscalls

I don't either, but in that case we should probably remove at least the synopsis (as the function signature listed there doesn't actually correspond to anything callable). Also, yeah, I guess documentation for the POSIX API would be nice, but since documentation already exists at opengroup (and there isn't anything in the serenity repository that is actually outdated) I wouldn't consider that a blocker.